scripts/release/workflow/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Set changeset status location
+# This is needed because `changeset status --output` only works with relative routes
+CHANGESETS_STATUS_JSON=$(node -p -e "require('path').relative(__dirname, '$RUNNER_TEMP/status.json')")
+
+# Save changeset status to temp file
+npx changeset status --output="$CHANGESETS_STATUS_JSON"
+
+# Get the list of releases from the status
+RELEASES=$(node -p -e "require('$CHANGESETS_STATUS_JSON').releases")
+
+# Defensive assertion. SHOULD NOT BE REACHED
+if [ $(node -p -e "$RELEASES.length") != 1 ]; then
+  echo "::error file=$CHANGESETS_STATUS_JSON::The status doesn't contain only 1 release"
+  exit 1;
+fi;
+
+# Get the next version
+NEW_VERSION=$(node -p -e "($RELEASES)[0].newVersion")
+
+# Create branch
+BRANCH_SUFFIX=$(echo $NEW_VERSION | awk -F'.' '{ print $1"."$2 }')
```

Comment:
Jq can simplify this a lot. It's available in GitHub Actions. The current code is not great IMO because we're basically eval-ing a string (the value of `$RELEASES`).

```suggestion
# Defensive assertion. SHOULD NOT BE REACHED
if [ "$(jq '.releases | length' "$CHANGESETS_STATUS_JSON")" != 1 ]; then
  echo "::error file=$CHANGESETS_STATUS_JSON::The status doesn't contain only 1 release"
  exit 1;
fi;

# Create branch
BRANCH_SUFFIX="$(jq -r '.releases[0].newVersion | gsub("\\.\\d+$"; "")')"
```